lib/internal/errors.js

```diff
@@ -674,6 +674,20 @@ const fatalExceptionStackEnhancers = {
   },
   afterInspector(error) {
     const originalStack = error.stack;
+    // Some consoles do not convert ANSI escape sequences to colors,
+    // rather display them directly to the stdout. On those consoles,
+    // libuv emulates colors by intercepting stdout stream and calling
+    // corresponding Windows API functions for setting console colors.
+    // However, fatal error are handled differently and we cannot easily
+    // highlight them. On Windows, detecting whether a console supports
+    // ANSI escape sequences is not reliable.
+    if (process.platform === 'win32') {
+      const info = internalBinding('os').getOSInformation();
+      const ver = info[2].split('.').map((a) => +a);
+      if (ver[0] !== 10 || ver[2] < 14393) {
+        return originalStack;
+      }
+    }
```

Comment:
Instead of adding this here, we could just change the following line: https://github.com/nodejs/node/blob/f446b2058dcfaf9d4e2f6bdfe555bc4437f41acf/lib/internal/tty.js#L150

It would have to be changed to `COLORS_2`.

Reply:
That would unnecessarily break valid `console.log` uses cases too.

Reply:
True, I guess in this case it's good as it is.